src/asf_enumeration/aria_s1_gunw.py

```diff
@@ -39,7 +39,7 @@ def does_intersect(self, geometry: shapely.Geometry) -> bool:
         Returns:
             does_intersect: if the frame instersects the geometry
         """
-        return shapely.intersects(self.polygon, geometry)
+        return bool(shapely.intersects(self.polygon, geometry))
```

Comment:
Was mypy not happy with shapely.intersects?

Reply:
Yes, and I confirmed it was a `numpy.bool` object at runtime.